src/app/home/home.component.scss

```diff
@@ -1,4 +1,22 @@
+@import 'variables';
+@import 'mixins';
+
 .home {
-  margin: 1em auto;
-  max-width: 1200px;
+  align-items: center;
+  align-content: center;
+  background: url("/assets/images/meadows.png");
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: $baseline * 10;
+  @include breakpoint(xs) {
```

Comment:
I'm not sure if the rule is enabled by default in the sass linter for this project, but I have always put mixins before declarations because of this:
https://github.com/sasstools/sass-lint/blob/master/docs/rules/mixins-before-declarations.md